quickwit-cli/src/index.rs

```diff
@@ -55,6 +55,20 @@ use crate::{
 pub fn build_index_command<'a>() -> Command<'a> {
     Command::new("index")
         .about("Create your index, ingest data, search, describe... every command you need to manage indexes.")
+        .subcommand(
+            Command::new("list")
+                .about("List indexes.")
+                .alias("ls")
+                .args(&[
+                    arg!(--config <CONFIG> "Quickwit config file")
+                        .env("QW_CONFIG"),
+                    arg!(--"data-dir" <DATA_DIR> "Where data is persisted. Override data-dir defined in config file, default is `./qwdata`.")
```

Comment:
Please remove the `data-dir` option, since this command should not read or write anything in the data dir.